modules/mono/editor/GodotTools/GodotTools/Build/DotNetFinder.cs

```diff
@@ -55,7 +56,8 @@ public static string FindDotNetExe()
             process.StartInfo = new ProcessStartInfo(dotNetExe, "--list-sdks")
             {
                 UseShellExecute = false,
-                RedirectStandardOutput = true
+                RedirectStandardOutput = true,
+                StandardOutputEncoding = Encoding.UTF8,
```

Comment:
It doesn't seem necessary here since we later set `DOTNET_CLI_UI_LANGUAGE` to `en-US` for this command.